api/v1/testkube.yaml

```diff
@@ -2100,6 +2100,34 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Problem"
+  /copy-files:
+    post:
+      tags:
+        - api
+        - tests
+        - executions
+        - test-suites
+      summary: "Upload file"
+      description: "Upload file to be copied into test suite executions"
+      operationId: uploadTestSuiteCopyFile
+      requestBody:
+        $ref: "#/components/requestBodies/CopyFileBody"
+      responses:
+        200:
+          description: "successful operation"
+          content:
+            text/yaml:
```

Comment:
is this json or yaml response

Reply:
it is `200`, but I'll add the json parts